sale_order_import_ubl/tests/files/UBL-Order-2.0-Example_no-party-name.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The module `sale_order_import_ubl` does not exist in `12.0` yet, here you are just adding this file and the one below, this is not correct.
You should wait for https://github.com/OCA/edi/pull/102 to be merged before proposing this change

Reply:
https://github.com/OCA/edi/pull/206#issuecomment-638769051